cmd/guaccollect/cmd/files.go

```diff
@@ -186,5 +191,15 @@ func initializeNATsandCollector(ctx context.Context, pubsubAddr string, blobAddr
 }
 
 func init() {
+	set, err := cli.BuildFlags([]string{"use-blob-url"})
```

Comment:
After the subsequent PRs that add this to other collectors, I imagine this will move to the top level command

Reply:
+1